init.lua

```diff
@@ -2,6 +2,9 @@ mail = {
 	-- api version
 	apiversion = 1.1,
```

Comment:
API has changed so should probably update this version number.

To follow semantic versioning should go through API changes to find out what new version should be.

Best would be to somehow document / summarize API changes, documentation of course is good but for pull request I think writing out summary of changes as PR comments / possibly including in PR description is most helpful.

Reply:
Ok, thx.

Reply:
This is still relevant, do you happen to have summary of API changes? (changed/removed/added function names, arguments and return values)
Of course I can also do this but it is a lot of API changes so I have to find some extra time for this.

I think for example mail.send arguments might not be compatible anymore but didn't check.